src/api/app/models/staging/staging_project.rb

```diff
@@ -105,11 +103,11 @@ def unassign_managers_group(managers)
     end
 
     def current_checks
-      relevant_checks + relevant_checks_for_architectures
```

Comment:
now that checks and status_reports are gone, can we remove all these current_ and relevant_ prefixes?